modules/server/src/network/aggregations/AggregationAccumulator.ts

```diff
@@ -1,41 +1,33 @@
 import { SUPPORTED_AGGREGATIONS } from '../common';
 import { Aggregations, Bucket, NumericAggregations } from '../types/aggregations';
 import { AllAggregations } from '../types/types';
-import { RequestedFieldsMap } from '../util';
 
 type ResolveAggregationInput = {
 	aggregationsMap: AllAggregations;
-	requestedAggregationFields: string[];
 	accumulator: AllAggregations;
 };
 
+type AggregationsTuple = [Aggregations, Aggregations];
+type NumericAggregationsTuple = [NumericAggregations, NumericAggregations];
+
 /**
  * Resolves returned aggregations from network queries into single accumulated aggregation
  *
  * @param
  */
-const resolveAggregations = ({
-	aggregationsMap,
-	requestedAggregationFields,
-	accumulator,
-}: ResolveAggregationInput) => {
-	requestedAggregationFields.forEach((fieldName) => {
+const resolveAggregations = ({ aggregationsMap, accumulator }: ResolveAggregationInput) => {
+	Object.keys(aggregationsMap).forEach((fieldName) => {
```

Comment:
iterate over response instead of request. possibility of undefined fields when using response object